### DISCUSSION TITLE
修改禁选状态下的提示信息

### DISCUSSION THREAD

**Discussion by 976868653** [ant-design/ant-design#33980] [Q&A] [answered]
请问下。在设置数据禁选后，禁选的这个图标是否支持修改，希望将这个图标修改为自定义提示文字
![image](https://user-images.githubusercontent.com/48016740/153139148-c3589281-28e5-4174-b718-88dad20d9dc3.png)

**Comment by yoyo837** [comment, marked as answer]
这是
```css
cursor: not-allowed;
```

的效果，你可以覆盖css

**Reply by 976868653** [reply]
感谢，你好，请问下我是需要覆盖哪个css文件的

**Reply by yoyo837** [reply]
请沟通您的开发人员处理。

**Reply by 976868653** [reply]
我就是开发，还在学习，这个是我直接将table内该行的数据 状态设置为disabled: true，然后系统框架的默认设置，不是我自己工程的css文件，所以不太清楚应该如何覆盖

**Reply by yoyo837** [reply]
这个抱歉了，既然在学习，就自己研究下吧。使用库的人我们都假设是掌握那些的人。
